gateway/tests/integration/routes_test.go

```diff
@@ -49,7 +49,7 @@ func fireRequestWithHeaders(url string, method string, reqBody string, headers m
 
 func TestGet_Rejected(t *testing.T) {
 	var reqBody string
-	unsupportedMethod := http.MethodHead
+	unsupportedMethod := http.MethodOptions
```

Comment:
Curious why this changed?

Reply:
reverted, both work. I was having some issues running the ci process.